src/librustc_codegen_ssa/back/linker.rs

```diff
@@ -898,6 +909,10 @@ impl<'a> Linker for EmLinker<'a> {
         });
     }
 
+    fn strip(&mut self) {
+        // noop
```

Comment:
For emscripten,  you should try to use `--llvm-opts` to pass `-strip-all` to the LLVM toolchain. See the [`emcc` documentation](https://emscripten.org/docs/tools_reference/emcc.html)

You may also need something like `-Wl` to make this work, I'm not sure.

Reply:
I didn't know if `--llvm-opts` can be specified multiple times. Can we just add two `--llvm-opts` and `['--strip-all']` here? Or we should add an field `llvmOpts` in `Emlinker`?